pkgs/development/libraries/qt-6/modules/qttranslations.nix

```diff
@@ -5,4 +5,10 @@
 qtModule {
   pname = "qttranslations";
   qtInputs = [ qttools ];
+  outputs = [ "out" ];
+
+  # FIXME: this should probably be lifted into qtModule
+  postFixup = ''
+    rm -rf $out/nix-support
+  '';
```

Comment:
What's this for?

Reply:
`qtModule` auto-propagates all of the dependencies, which is how a lot of things ended up depending on qttranslations when they really wanted qttools. This just nukes the propagated deps so it's harder to make the same mistake in new packages. I do still want to move this into qtModule as an optional flag or something but this worked well enough for a PoC.

Reply:
We should drop the `qtModule` shorthand one day. Can we use `buildInputs` here?

Reply:
> We should drop the `qtInputs` shorthand one day. Can we use `buildInputs` here?

https://github.com/NixOS/nixpkgs/blob/c5b5fad9134ed3b523a0e28d5ee554af8de9d3af/pkgs/development/libraries/qt-6/qtModule.nix#L25

It's just another name for `propagatedBuildInputs` anyway.

Reply:
Now that I think about it, qttools should really be in `nativeBuildInputs` even.

Reply:
Let's try it!

Reply:
I am trying it right now :)

Reply:
Took a little tweak to qtModule, but it did work!